test/kube2e/gateway/gateway_test.go

```diff
@@ -1451,7 +1451,7 @@ var _ = Describe("Kube2e: gateway", func() {
 				}
 				return eris.Errorf("expected all test resources to have been deleted but found: "+
 					"%d pods, %d virtual services, %d proxies", len(coloredPods.Items), len(vsList), len(proxyList))
-			}, time.Minute, time.Second).Should(BeNil())
+			}, time.Minute, time.Second).Should(Succeed())
```

Comment:
Why does this change?

Reply:
Linter error.

Reply:
The `[ginkgolinter](https://golangci-lint.run/usage/linters/#ginkgolinter)` has a `force-succeed` option that is new and on by default.

Reply:
ty!